src/pages/Home/component/LandingFeature.jsx

```diff
@@ -0,0 +1,12 @@
+const LandingFeature = ({ featureCardImg, featureCardName }) => {
+  return (
+    <div className="home_features">
+      <img className="featureImg" src={featureCardImg} alt="image" />
```

Comment:
Try to give more descriptive values to alt

Reply:
thank you sudhanshu i will keep it in mind.